README.md

```diff
@@ -1261,6 +1261,12 @@ The examples of decoupled models shows how to develop and serve
 You can find the complete example instructions in
 [examples/decoupled](examples/decoupled/README.md).
 
+## Model Instance Kind
+
+Triton allows an execution of a model to be performed on both CPU, and GPU. 
```

Comment:
Triton model configuration allows users to provide kind to instance group settings<link:https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_configuration.md#instance-groups>. A python backend model can be written to respect the kind setting to control the execution of a model instance either on CPU or GPU.

In the ... we demonstrate how this can be achieved for your python model.